Documentation/BuildAndDeploy.md

```diff
@@ -0,0 +1,16 @@
+# Building and Deploying MRTK
+When you're ready to run your app on your device a standalone app (for HoloLens, Android, iOS, etc.), you need to build and deploy your project. Building an deploying an app that uses MRTK is just like building and deploying any other Unity app. There are no MRTK-specific instructions. Read below for detailed steps on how to build and deploy a Unity app for HoloLens.  Learn more about building for other platforms at [Publishing Builds](https://docs.unity3d.com/Manual/PublishingBuilds.html).
+
+### Building and Deploying MRTK to HoloLens 1 and HoloLens 2 (UWP)
+As mentioned above, building an app that uses MRTK is just like building any other UWP app, there's nothing MRTK specific. You can follow instructions at [building your application to device](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base-ch1#build-your-application-to-your-device)  for more information on how to build and deploy your app to HoloLens.
```

Comment:
```suggestion
Instructions on how to build and deploy for Hololens 1 and Hololens 2 (UWP) can be found at [building your application to device](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base-ch1#build-your-application-to-your-device) .
```

i don't think we have to refer to the previous paragraph , it doesn't add anything here? what do you think?